CHANGELOG.md

```diff
@@ -1,16 +1,24 @@
 ## vNext (TBD)
 
+### Breaking Changes
```

Comment:
We should not be introducing breaking changes as we already discussed

Reply:
we should remove this from the changelog

Reply:
I will move them to `Enhancements`.